product_brand_stock/i18n/es.po

```diff
@@ -4,7 +4,7 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0\n"
+"Project-Id-Version: Odoo Server 15.0\n"
```

Comment:
You're changing this one but not modifying the template.

Either change bot or leave both untouched (they will be updated by the bot anyway).